homeassistant/components/ads/sensor.py

```diff
@@ -41,27 +44,31 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     name = config[CONF_NAME]
     unit_of_measurement = config.get(CONF_UNIT_OF_MEASUREMENT)
     factor = config.get(CONF_ADS_FACTOR)
+    digit = config.get(CONF_ADS_DIGIT)
```

Comment:
Please use `dict[key]` for required config keys and keys with default schema values.
```suggestion
    digit = config[CONF_ADS_DIGIT]
```

Reply:
fixed.